pkg/kubelet/prober/worker.go

```diff
@@ -90,6 +92,7 @@ func newWorker(
 		container:       container,
 		probeType:       probeType,
 		probeManager:    m,
+		randomFloat:     randomFloat, // rand.Float64() used to calculate sleep duration on kubelet start
```

Comment:
I think the code will be more predictable if the jitter value calculated as a random interval of a minimal between startup, readiness, and liveness is used by all workers for a single container. i don't think it is a big deal to start three workers at the same time, important to have different jitter value for different containers.